dash/dash.py

```diff
@@ -1014,17 +1062,45 @@ def get_asset_url(self, path):
 
     def enable_dev_tools(self,
                          debug=False,
-                         dev_tools_serve_dev_bundles=None):
+                         dev_tools_serve_dev_bundles=None,
+                         dev_tools_hot_reload=None,
+                         dev_tools_hot_reload_interval=None,
+                         dev_tools_hot_reload_watch_interval=None,
+                         dev_tools_hot_reload_max_retry=None,
+                         dev_tools_silence_routes_logging=None):
         """
         Activate the dev tools, called by `run_server`. If your application is
         served by wsgi and you want to activate the dev tools, you can call
         this method out of `__main__`.
 
-        :param debug: If True, then activate all the tools unless specified.
+        If an argument is not provided, it can be set with environ variables.
+
+        Available dev_tools environ variables:
+
+        :param debug: If True, then activate all the tools unless specifically
+            disabled by the arguments or by environ variables. Available as
+            `DASH_DEBUG` environ var.
         :type debug: bool
-        :param dev_tools_serve_dev_bundles: Serve the dev bundles.
+        :param dev_tools_serve_dev_bundles: Serve the dev bundles. Available
+            as `DASH_SERVE_DEV_BUNDLES` environ var.
         :type dev_tools_serve_dev_bundles: bool
-        :return:
+        :param dev_tools_hot_reload: Activate the hot reloading. Available as
+            `DASH_HOT_RELOAD` environ var.
+        :type dev_tools_hot_reload: bool
+        :param dev_tools_hot_reload_interval: Interval at which the client will
+            request the reload hash. Available as `DASH_HOT_RELOAD_INTERVAL`
+            environ var.
+        :type dev_tools_hot_reload_interval: int
+        :param dev_tools_hot_reload_watch_interval: Interval at which the
+            assets folder are walked for changes. Available as
+            `DASH_HOT_RELOAD_WATCH_INTERVAL` environ var.
+        :type dev_tools_hot_reload_watch_interval: float
+        :param dev_tools_hot_reload_max_retry: Maximum amount of retries before
```

Comment:
Were these two (dev_tools_hot_reload_max_retry, dev_tools_silence_routes_logging) purposefully omitted from the variables you can set using the environment? It would be nice to have the full functionality via either method.